docs/sdk/cosmos-sdk-cli.md

```diff
@@ -0,0 +1,34 @@
+# cosmos-sdk-cli 
+Create a new blockchain project based on cosmos-sdk with a single command.
+
+---
+
+# Installation
+
+```shell
+go get github.com/cosmos/cosmos-sdk
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+make all
```

Comment:
Can you switch to this to the minimal make command required?

Reply:
Yes!